src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,38 @@
 
 class Animal {
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
This should be a static property

Reply:
If making it static causes some linter errors you don't have to change it ;)
Sorry, I've just realized ;)

Reply:
I managed to change linter options

Reply:
Nevermind, not really. Tests in VS code passed, not on github however, I've changed it back